src/Draco.Compiler/Internal/Binding/Binder_UntypedExpression.cs

```diff
@@ -469,27 +474,64 @@ private UntypedExpression BindMemberExpression(MemberExpressionSyntax syntax, Co
             // Error, don't cascade
             return new UntypedReferenceErrorExpression(syntax, err.Symbol);
         }
-        else if (left is UntypedModuleExpression moduleExpr)
+        Symbol? type = left is UntypedModuleExpression
```

Comment:
Since this is becoming a pattern (checking for module _or_ type on the left), can you think of some nice way to factor these together?